src/PackagingProjects/BuildAgentPayload.csproj

```diff
@@ -15,26 +15,63 @@
     <!-- OutputPath is required by the Clean and Rebuild tasks -->
     <OutputPath>bin\$(Configuration)\</OutputPath>
   </PropertyGroup>
+
+  <!-- Binaries for dotnet framework scanner -->
   <ItemGroup>
-    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\net46\SonarQube.Scanner.MSBuild.exe" />
-    <FilesToCopy Include="$(SourcesRoot)\SonarQube.Old.Bootstrapper\bin\$(Configuration)\net46\MSBuild.SonarQube.Runner.exe" />
     <FilesToCopy Include="$(SourcesRoot)\SonarQube.Bootstrapper\bin\$(Configuration)\net46\SonarQube.Analysis.xml" />
```

Comment:
Ideally this file would use the build variables defined in _Directory.Build.props_ e.g. $(ScannerNetFxVersion).
We can do that later after the changes have been merged to master.